org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/preferences/LanguageRootPreferencePage.java

```diff
@@ -21,6 +21,13 @@
 	private @Inject ResetPromptDialogSettingsField resetPromptDialogSettings;
 
 	@Override
+	public void createControl(Composite parent) {
+		super.createControl(parent);
+		getApplyButton().setVisible(false);
```

Comment:
which sense do they make on the root page? They would make sense if we could reset all sub-pages, but I did not find a way to get the sub-pages of LanguageRootPreferencePage.

<img width="621" alt="screenshot 48" src="https://user-images.githubusercontent.com/265597/28009736-cdf6e1b8-655c-11e7-956e-b6ac87079bc4.png">